src/utils/api/api.js

```diff
@@ -0,0 +1,68 @@
+import  {fetcher} from './fetcher'
+import AsyncStorage from '@react-native-async-storage/async-storage';
+export class Api {
+  constructor(host) {
+    this.host = host;
+    this.api = `${this.host}/api`;
+  }
+
+  categories = async () => {
+    return fetcher(`${this.api}/categories?populate=*`);
+  };
```

Comment:
why this is just called categories, whereas request to get bookmarks is called getBookmarks? Maybe getCategories?